moxy/org.eclipse.persistence.moxy/src/main/java/org/eclipse/persistence/jaxb/dynamic/metadata/Metadata.java

```diff
@@ -28,9 +28,10 @@
     protected Map<String, XmlBindings> bindings;
     protected DynamicClassLoader dynamicClassLoader;
 
+    @SuppressWarnings("unchecked")
     public Metadata(DynamicClassLoader dynamicClassLoader, Map<String, ?> properties) {
         this.dynamicClassLoader = dynamicClassLoader;
-        this.bindings = JAXBContextFactory.getXmlBindingsFromProperties(properties, dynamicClassLoader);
+        this.bindings = JAXBContextFactory.getXmlBindingsFromProperties((Map<String, Object>) properties, dynamicClassLoader);
```

Comment:
if constructor takes string, Object then the cast is not needed here

Reply:
This changeset was removed from PR.